### ISSUE TITLE
deleteOnFileDelete:true marks files as deleted if deleted from Windows Explorer

### ISSUE THREAD

**Issue by maxnoel** [stef-levesque/vscode-perforce#68]
I'm using VSCode 1.11.2 with vscode-perforce 2.0.1.

Reproduction case:

- Enable perforce.deleteOnFileDelete. Restart VS Code to ensure the option is honored.
- Open a folder that's tracked by Perforce with VSCode. Expand the tree view to reveal files.
- Also open that folder in the Windows Explorer.
- From the Windows Explorer window, delete one file that's visible in VSCode's tree view.
- Observe that the file was marked as deleted in the P4 workspace's default changelist.

This might be intended behavior, but I'd only expect vscode-perforce to mark as deleted the files that I'm explicitly deleting from within VSCode (especially since re-creating those files does *not* re-add them).

**Comment by stef-levesque** [comment]
cc @jel-massih

**Comment by jel-massih** [comment]
Yeah it uses a filesystemlistener so anything that affects the files while extension is running will get picked up.

I think a better course would to be more naturally handle delete/add (so if a file is marked for delete then re-added, it would be marked for edit).
